packages/reference/src/common/MultipleReferenceEditor.tsx

```diff
@@ -61,30 +62,32 @@ function Editor(props: EditorProps) {
     );
   }, [props.items]);
 
-  const onSortStart: SortStartHandler = useCallback((_, event) => {
-    if (event instanceof MouseEvent) {
-      document.body.classList.add('grabbing');
-    }
-    event.preventDefault();
+  const { rearrangeSortIDs } = useSortIDs(items);
+
+  const onSortStart = useCallback(() => {
+    document.body.classList.add('grabbing');
```

Comment:
Why did we remove `if (event instanceof MouseEvent) {`
I guess sorting can also be done with the keyboard, maybe then we don't want to have the `cursor: grabbing` ?

Reply:
`DragStartEvent` do not extend MouseEvent